uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/leave-type-wizard.constants.js

```diff
@@ -216,5 +221,19 @@ define([
           }
         ]
       }
-    ]);
+    ])
+    .constant('leave-type-categories-icons', {
+      leave: 'plane',
+      sickness: 'medkit'
+    })
+    .constant('leave-type-categories-hidden-tabs-by-category', {
```

Comment:
i'm having a hard time deciphering this name, you mean something like "tabs-hidden-by-category"? In any case, the `leave-type-categories` prefix looks redundant

Reply:
Done.